### ISSUE TITLE
Strange messages in the status bar

### ISSUE THREAD

**Issue by coolvitto** [x64dbg/Translations#4]
Some of the debug menus have strange status bar messages when you hover over them.
The snapshot being used is "snapshot_2024-08-26_18-37".

![snp-5](https://github.com/user-attachments/assets/82821500-6a7e-48ff-af91-8aec5da0786a)

**Comment by mrexodia** [comment]
These are not triggered by the hovering, it's the message explaining why the program has paused (there was an int3 breakpoint in gdi32full.dll).

**Comment by coolvitto** [comment]
Do such messages appear even if debugging has not started?

**Comment by mrexodia** [comment]
Ah you're right, there is a description field for some of the actions:

![image](https://github.com/user-attachments/assets/dc3035a3-226b-4fbe-abce-740f1bc9939a)

**Comment by coolvitto** [comment]
Yes.
Most menus have explanatory text for their commands, but some have strange messages.

**Comment by mrexodia** [comment]
I think some just do not have any messages and then it displays whatever was displayed before?

**Comment by coolvitto** [comment]
This problem doesn't occur when I don't have any files open, but it seems to occur when I open a file.
![snp-7](https://github.com/user-attachments/assets/6cca9884-9a49-4998-aa77-a3919ba3a0d9)

I've tried a few other languages, but this problem seems to only apply to Japanese.

My guess is that this problem occurs with menu commands that don't have a message to display in the status bar.
I tried changing the language to English, but even in English there is no status bar message for this menu command.
![snp-6](https://github.com/user-attachments/assets/8663ed6e-73e6-4cc4-9fdb-c9e2b0631bc4)

**Comment by mrexodia** [comment]
Yeah so there is no issue? Those commands just don't have a description, but once the description is added it will show up in the translations automatically!

**Comment by coolvitto** [comment]
That's true.
I don't think it's something to worry about for everyday use, so I'll wait and see how it goes.
